### ISSUE TITLE
Swipe to change track on Music plugin

### ISSUE THREAD

**Issue by gtataranni** [Toxblh/MTMR#235]
[enhancement] Similarly to what happens on smartphones touchscreens, a single-finger swipe could be used to go to previous/next track on the music player (Spotify or iTunes)

**Comment by jkhdesign** [comment]
I agree! I think it would be more intuitive if tapping the track paused it and swiping played the next/previous track.

**Comment by dyraosx** [comment]
https://github.com/Toxblh/MTMR/issues/252
